src/Selenium2Library/keywords/_element.py

```diff
@@ -680,6 +679,31 @@ def remove_location_strategy(self, strategy_name):
         """
         self._element_finder.unregister(strategy_name)
 
+    def element_attribute_should_be(self, locator, attr):
+        """Checks whether element has attribute with value
+
+        Usage Example:
+        | Element Attribute Should Be | xpath=//input | key=value |
+        """
+        self._info("Checking attributes '%s' for element '%s'" % (
+                                                    attr, locator))
+        try:
+            key, value = attr.split("=", 1)
+        except ValueError:
+            raise AssertionError(
+                    "Attribute %s does not match pattern key=value" % attr)
+
+        el = self._element_find(locator, True, True)
+        retval = el.get_attribute(key)
+
+        if retval is None:
```

Comment:
The ${EMPTY} and ${None} are, at least for me, the same thing, perhaps different sides of the same coin. For me they both mean the same thing, nothing. Also I would not like to limit the users to use specific kind of variable, ${EMPTY}, just to verif  that attribute don't have value.

Also from the code maintainability point of view, doing ```if retval is None:``` causes a problem because it's against [idiomatic Python truth value testing](http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html#testing-for-truth-values). 

The later is not that important if there is a good usecase for preventing using ```key=${None}``` to check that attribute don't have value. Could you define a use case where it's beneficial to prevent the None?